paimon-core/src/main/java/org/apache/paimon/table/AbstractFileStoreTable.java

```diff
@@ -164,6 +164,14 @@ public Identifier identifier() {
                 : identifier;
     }
 
+    @Override
+    public String uuid() {
+        if (catalogEnvironment.uuid() != null) {
+            return catalogEnvironment.uuid();
+        }
+        return fullName();
```

Comment:
should at least log warning or add a setting to enabling simply use `fullName()` as `uuid()`